index.bs

```diff
@@ -2778,6 +2778,52 @@ enum ChannelInterpretation {
 </table>
 </div>
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn>. This means that even when the
+{{AudioNode}} is fed silence, non-silent output can be output.
+
+{{AudioNode}}s have a non-zero tail-time if they have internal processing state
+such that input received at one time affects the output at later times. Such
+{{AudioNode}}s that have received non-silent input continue to produce
+non-silent output for the length of the tail-time after the input becomes
+silent.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>, if any of the following conditions hold.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] if and only if it
+	is playing for at least part of the current rendering quantum.
+- A {{MediaElementAudioSourceNode}} is [=actively processing=] if and only if its
+	{{MediaElementAudioSourceNode/mediaElement}} is playing for at least part of
+	the current rendering quantum.
+- A {{MediaStreamAudioSourceNode}} or {{MediaStreamTrackAudioSourceNode}} when
+	the [[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] used as input
+	has a <code>readyState</code> of <code>live</code>, a <code>muted</code>
+	attribute that is <code>false</code> and an <code>enabled</code> attribute
+	that is <code>true</code>.
+- A {{DelayNode}} in a cycle is [=actively processing=] only when the absolute value
+	of any output sample for the current [=render quantum=] is greater than or equal
+	to \( 2^{-126} \).
```

Comment:
This is a really, really tight constraint.  Let's say you have a biquad filter in the loop.  It's possible, due to round-off that the biquad never produces 0 output for 0 input.  

Not sure if this is achievable in practice.

Reply:
I wonder whether you may have missed my reply to your previous comment about this.
https://github.com/WebAudio/web-audio-api/pull/1810#discussion_r253250386

The reason for choosing FLT_MIN here rather than FLT_TRUE_MIN is that the output does not need to reach zero.

Because full precision is available at FLT_MIN, the round-off occurring with values around FLT_MIN is similar to round-off at larger values.  If the decay (biquad or other) is such that input of any magnitude will attenuate, then values around FLT_MIN will attenuate.  After values around FLT_MIN attenuate, the cutoff is passed and the node outputs a single channel of silence.

Regardless of the value of the cutoff, perhaps there may be a chance of instability due to the sharp gradient at the cutoff if a biquad does not attenuate that sufficiently.  I don't know whether or not the output from such a biquad would decay to zero without the cutoff.  Even if the cutoff is introducing some new instability, it would be preferable that any remaining non-zero output from this corner-case would be determined by the precision of float32 rather than an arbitrary cutoff.

Reply:
This is acceptable to me. I still find the constraint very tight.  Does Firefox stop a BiquadFilter tail when it reaches FLT_MIN?  Chrome just estimates the tail time by computing when the impulse response is about 60 dB down (I think).  Even then, there are filters whose tail time is in the thousands of seconds.

Reply:
I wouldn't say RT60 is an arbitrary cutoff. It is considered the industry standard for reverberation decay. I think it has been accepted as a reasonable threshold for "perceptually inaudible" sound.

Can we make it up to UA? I think the numeric difference between RT60 and FLT_MIN is quite significant, but the audible difference would not really big. In that case, UA might be able to try different optimization techniques to improve the rendering performance.